### ISSUE TITLE
Error for some buttons: No device ID currently in memory

### ISSUE THREAD

**Issue by JLester73** [iamtheyammer/gam-cros-win-wrapper#11]
The program looks great and will definitely save some time.  I can search for devices and it find them and shows the info in the output window.  Some buttons though give the error: "No device ID currently in memory. Press Submit then try again."  The attached screenshots show me searching for a device by asset id, then clicking the "Set Location" button.  Set Asset ID works, Set User gives the same error.

![2018-10-10 15_33_09-releases jay0lee_gam github](https://user-images.githubusercontent.com/22177857/46761454-8270d100-cca2-11e8-8621-5380fdf2948b.png)
![2018-10-10 15_35_19-releases jay0lee_gam github](https://user-images.githubusercontent.com/22177857/46761460-84d32b00-cca2-11e8-86ae-6f43d1ae3e26.png)

**Comment by iamtheyammer** [comment]
Ok, so it looks like you're searching by Asset ID here. One thing I recommend is that if you search by asset ID a lot, go into File -> Preferences and turn on the first tick box (something like prioritise asset ID over serial number during search). It should speed up your searches by about 30%.

On to your error, I'm wondering if there are two devices with that Asset ID?

**Comment by iamtheyammer** [comment]
(my bad!)

**Comment by iamtheyammer** [comment]
Oh, and I should mention you can find the answer to my question by running `gam print cros query "asset_id:130070"` and seeing if two device IDs pop out

**Comment by JLester73** [comment]
Definitely just one device with that ID.  I tried a few others by serial as well and get the same error, still only for certain buttons though.  I did just upgrade to GAM 4.6.1, but it's been out for a while I think.

**Comment by iamtheyammer** [comment]
That's super funky. Here's what I did wrong:  
ChromebookGUI/ChromebookGUI/MainWindow.xaml.cs lines 103, 163, 189 contain the OR (`||`) operator not the and (`&&`) operator. I will rebuild this in about an hour, or would it help you if I did it now?

**Comment by JLester73** [comment]
It's no big deal for me, so no hurry.  Glad it was an easy fix.

**Comment by iamtheyammer** [comment]
Yeah that's another ID10T error. I'll get it done for you in a little while
